README.rst

```diff
@@ -38,21 +38,21 @@ such as emulators, virtual machines and simulation tools.
 
 The SIL Kit ecosystem comprises the following turn-key solutions:
 
-* The `SIL Kit Adapters for QEMU <https://github.com/vectorgrp/sil-kit-adapters-qemu>`_
+* The `SIL Kit Adapter for QEMU <https://github.com/vectorgrp/sil-kit-adapters-qemu>`_
   integrates with QEMU to support co-simulation with emulated targets.
 
-* The `SIL Kit Adapters for TAP devices <https://github.com/vectorgrp/sil-kit-adapters-tap>`_
+* The `SIL Kit Adapter for TAP devices <https://github.com/vectorgrp/sil-kit-adapters-tap>`_
   project provides first-class support for TAP devices of the host operating system.
 
-* The `SIL Kit Adapter for SocketCAN <https://github.com/vectorgrp/sil-kit-adapters-vcan>`_
+* The `SIL Kit Adapter for virtual CAN <https://github.com/vectorgrp/sil-kit-adapters-vcan>`_
   can be used to attach a virtual CAN (Controller Area Network) interface (SocketCAN) to a Vector SIL Kit CAN bus.
```

Comment:
I suggest we remove writing out the CAN abbrevation. If you read until here, you probably know it is and it looks odd with the (SocketCAN) afterwards.